packages/compass-crud/src/components/document-list.less

```diff
@@ -154,3 +154,9 @@
     }
   }
 }
+
+.loader {
```

Comment:
just curious, why was this necessary?

Reply:
nice. the `CancelLoader` component previously centered the content and also added `height: 100%`.  I removed that and made component occupy width/height it needed and let consumer of the component style it as needed.